apps/minifront-v2/app/v2.css

```diff
@@ -0,0 +1,106 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* Font face definitions */
+@font-face {
```

Comment:
issue: these styles should all come from UI package. When you build it, it produces `'@penumbra-zone/ui/style.css'` with fonts built into it. The same is about scrollbars in the end of this file – i think the file should be safe to delete

Reply:
this allows removing font files from minifront v2 too